bus-mapping/src/evm/opcodes/pop.rs

```diff
@@ -0,0 +1,146 @@
+use core::ops::Deref;
+use super::Opcode;
+use crate::{
+    exec_trace::{ExecutionStep, TraceContext},
+    operation::{StackOp, RW},
+    Error,
+};
+
+/// Placeholder structure used to implement [`Opcode`] trait over it corresponding to the
+/// [`OpcodeId::POP`](crate::evm::OpcodeId::POP) `OpcodeId`.
+/// This is responsible of generating all of the associated [`StackOp`]s and place them
+/// inside the trace's [`OperationContainer`](crate::operation::OperationContainer).
+#[derive(Debug, Copy, Clone)]
+pub(crate) struct Pop;
+
+impl Opcode for Pop {
+    fn gen_associated_ops(
+        &self,
+        ctx: &mut TraceContext,
+        // Contains the POP instr and the value which is last placed in the stack.
+        exec_step: &mut ExecutionStep,
+        // Contains the next step.
+        next_steps: &[ExecutionStep],
+    ) -> Result<(), Error> {
+        ctx.push_op(
+            exec_step,
+            StackOp::new(
+                RW::READ,
+                // Get the value and addr from the execution step and reads the value of the last filled stack element
+                exec_step.stack().last_filled(),
+                exec_step.
+                    .stack()
+                    .deref()
+                    .last()
+                    .cloned()
```

Comment:
Try `copied`. It might work since `EvmWord` is an array and should be copiable.

Reply:
I still didnt understand this